app/controllers/concerns/base_crud_controller.rb

```diff
@@ -0,0 +1,275 @@
+##
+## provides genericized view handling of the various model classes
+##
+## can scoped to given base namespace and view path (assigned by Helper)
+##
+
+
+#todo: consider base class vs mixin
+class BaseCrudController < ApplicationController
+  include BaseCrudHelper
+
+  layout 'raw_crud'
+
+
+  def clazz
+    raise "not implemented - clazz"
```

Comment:
You could implement this as `controller_name.classify.constantize`, which will raise a `NameError` on failure, and it would let you drop all the overrides in each of the other controllers. Not a big deal, but it's also a pretty easy and low-impact change.